src/containerapp/azext_containerapp/tests/latest/test_containerapp_session_code_interpreter_nodelts.py

```diff
@@ -0,0 +1,131 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import os
+from azure.cli.command_modules.containerapp._utils import format_location
+
+from azure.cli.testsdk.scenario_tests import AllowLargeResponse
+from azure.cli.testsdk import (ScenarioTest, ResourceGroupPreparer, JMESPathCheck)
+
+from .common import (TEST_LOCATION)
+from .utils import create_containerapp_env
+
+TEST_DIR = os.path.abspath(os.path.join(os.path.abspath(__file__), '..'))
+
+class ContainerAppSessionCodeInterperterNodeLTSTests(ScenarioTest):
+    @AllowLargeResponse(8192)
+    @ResourceGroupPreparer()
+    def test_containerapp_session_code_interpreter_nodelts_e2e(self, resource_group):
+        location = TEST_LOCATION
+        self.cmd('configure --defaults location={}'.format(location))
+
+        env_name = self.create_random_name(prefix='aca-sp-env', length=24)
+        create_containerapp_env(self, env_name, resource_group, TEST_LOCATION)
```

Comment:
why do you need to create env?

Reply:
Hi @ASHIQUEMD 
The `env_name` is unused, please remove it is  not necessary:
The code about creating env at `test_containerapp_session_code_interpreter` also not needed
https://github.com/Azure/azure-cli-extensions/blob/f0677b54631ebd34e6bae5b9643234bcd59f1dfe/src/containerapp/azext_containerapp/tests/latest/test_containerapp_session_code_interpreter.py#L24